### PR TITLE
Document options for php and js code formatting tools

### PR DESCRIPTION
**Pull request by ChristophWurst**
Ref https://github.com/nextcloud/server/pull/20209

### CODE REVIEW

File: developer_manual/app/coding-style.rst [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+PHP
+---
+
+Starting with Nextcloud 19 there is a shared `PHP Coding Standards Fixer <https://github.com/FriendsOfPhp/PHP-CS-Fixer>`_ configuration you can use to automatically format your app's source code. For full details see the `repository on Github <https://github.com/nextcloud/coding-standard/>`_.
+
+JavaScript
+----------
+
+There is a shared configuration for `eslint <https://eslint.org/>`_ that you can use to automatically format your Nextcloud apps's JavaScript code. It consists of two parts: a `config package <https://github.com/nextcloud/eslint-config>`_ that contains the formatting preferences and a `plugin <https://github.com/nextcloud/eslint-plugin>`_ to detect deprecated and removed APIs in your code. See their readmes for instructions.
```
Comment (skjnldsv) [inline developer_manual/app/coding-style.rst:15]: Maybe add a default .eslintrc.js ?

Comment (skjnldsv) [reply developer_manual/app/coding-style.rst:15]:
```js
// .eslintrc.js 
module.exports = {
	extends: [
		'@nextcloud',
	]
}
```

Comment (ChristophWurst) [reply developer_manual/app/coding-style.rst:15]: If you have one please share it!

Comment (ChristophWurst) [reply developer_manual/app/coding-style.rst:15]: it should be in the repos, though

Comment (skjnldsv) [reply developer_manual/app/coding-style.rst:15]: I should not have approved, there was one example here in the comments :p

Comment (ChristophWurst) [reply developer_manual/app/coding-style.rst:15]: still. it links the repos. it should be there and only there. anyhting that is duplicated will get out of sync

Comment (skjnldsv) [reply developer_manual/app/coding-style.rst:15]: You're right :)

**Pull request merged**